topics/fair/tutorials/data-management/tutorial.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: tutorial_hands_on
+title: FAIR data management solutions
+zenodo_link: ''
+questions:
+- Is there a reproducibility crisis?
+- What can go wrong with data analysis?
+objectives:
+- Learn best practices in data management 
+- Learn how to introduce computational reproducibility in your research
+time_estimation: 10m
+key_points:
+- FAIR data management allows machines to automatically find and use the data accordingly.
+
+subtopic: fair-data
+
+requirements:
+  -
+    type: "internal"
+    topic_name: fair
+    tutorials:
+      - fair-intro
+      
+tags:
+  - fair
+  - dmp
+  - data stewardship
+  
+contributors:
+- kkamieniecka
+- poterlowicz-lab
```

Comment:
If you like can use our fancy new contributions syntax to more accurately record contributions to a training material, declaring which people should be recorded for authorship or just editing.

```
contributions:
  authorship:
    - shiltemann
  editing:
    - bebatut
    - hexylena
  funding:
    - carpentries
  testing:
    - userX
  ux:
    - userY
  infrastructure:
    - userZ
```

see the [full gtn tutorial schema documentation](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/schemas/tutorial.html) for details on the individual roles.

Reply:
sounds good, I will add you as a fairy godmother of my training 👍